cms/apps/media/templates/admin/media/includes/focal-point.html

```diff
@@ -0,0 +1,79 @@
+<fieldset class="module aligned with-legend ">
+  <h2 class="legend">Focal point</h2>
+  <div>Specifying a focal point for an image should prevent that location from being cropped.</div>
+  <div id="focal-point-clear">Clear</div>
+
+  <div id="focal-point-container" style="position: relative; width: fit-content">
+    <img id="focal-point" style="filter:{% if original.focal_y %}brightness(50%){% else %}brightness(100%){% endif %};" src="{{ original.get_absolute_url }}"/>
+    <div id="focal-point-pointer"></div>
+  </div>
+  <script>
+    (function () {
+      document.addEventListener('DOMContentLoaded', function() {
+        document.getElementById('focal-point').addEventListener('click', function (e) {
+          var pointer = document.getElementById('focal-point-pointer')
+          var image = document.getElementById('focal-point')
+
+          var imageW = image.offsetWidth;
+          var imageH = image.offsetHeight;
+
+          //Calculate FocusPoint coordinates
+          var offsetX = e.pageX - image.getBoundingClientRect().left;
+          var offsetY = e.pageY - image.getBoundingClientRect().top;
+
+          //Calculate CSS Percentages
+          var percentageX = (offsetX/imageW)*100;
+          var percentageY = (offsetY/imageH)*100;
+
+          pointer.style.left = ((offsetX - (pointer.offsetWidth/2))/imageW)*100 + '%'
+          pointer.style.top = ((offsetY - (pointer.offsetHeight/2))/imageH)*100 + '%'
+          pointer.style.display = 'block'
+          image.style.filter = 'brightness(50%)'
+
+          document.getElementById('id_focal_x').value = percentageX
+          document.getElementById('id_focal_y').value = percentageY
+        })
+        document.getElementById('focal-point-clear').addEventListener('click', function (e) {
+          document.getElementById('id_focal_x').value = ''
+          document.getElementById('id_focal_y').value = ''
+          document.getElementById('focal-point').style.filter = 'brightness(100%)'
+          document.getElementById('focal-point-pointer').style.display = 'none'
+        })
+      })
+    })()
+  </script>
+  <style>
+    #focal-point-pointer {
+      position: absolute;
+      top: {% if original.focal_y %}calc( {{ original.focal_y }}% - 5px){% else %}50%{% endif %};
```

Comment:
I think the `{% else %}` needs to be moved to the left a bit - currently if there's no `original.focal_y` it will be misplaced by 5 pixels (it should be `calc(50% - 5px)`).